tracer/test/Datadog.Trace.Tests/LambdaCommonTests.cs

```diff
@@ -23,73 +23,32 @@ public class LambdaCommonTests
 
         [Fact]
         [Trait("Category", "ArmUnsupported")]
-        public void TestCreatePlaceholderScopeSuccess()
+        public void TestCreatePlaceholderScopeSuccessWithoutContext()
         {
-            var response = new Mock<HttpWebResponse>(MockBehavior.Loose);
-            response.Setup(c => c.StatusCode).Returns(HttpStatusCode.OK);
-            response.Setup(c => c.Headers.Get("x-datadog-trace-id")).Returns("1111");
-            response.Setup(c => c.Headers.Get("x-datadog-span-id")).Returns("2222");
-
-            var httpRequest = new Mock<WebRequest>();
-            httpRequest.Setup(h => h.GetResponse()).Returns(response.Object);
-
-            _lambdaRequestMock.Setup(lr => lr.GetTraceContextRequest()).Returns(httpRequest.Object);
-
             var tracer = TracerHelper.Create();
-            var scope = LambdaCommon.CreatePlaceholderScope(tracer, _lambdaRequestMock.Object);
-
+            var scope = LambdaCommon.CreatePlaceholderScope(tracer, null, null);
             scope.Should().NotBeNull();
-            scope.Span.TraceId.ToString().Should().Be("1111");
-            scope.Span.SpanId.ToString().Should().Be("2222");
+            scope.Span.TraceId.ToString().Should().NotBeNull();
+            scope.Span.SpanId.ToString().Should().NotBeNull();
+            scope.Span.Context.TraceContext.SamplingPriority.Should().Be(1);
         }
 
         [Fact]
         [Trait("Category", "ArmUnsupported")]
-        public void TestCreatePlaceholderScopeFailureDueToInvalidLong()
```

Comment:
Can you re-add a test for the scenario where the input `traceId` string is not a valid long? I suppose you could also do this with an invalid `samplingPriority` string too

Reply:
✅